packages/modeling/src/operations/extrusions/slice/equals.test.js

```diff
@@ -0,0 +1,17 @@
+const test = require('ava')
+
+const { equals, fromPoints } = require('./index')
+
+test('slice: equals() should return proper value', (t) => {
+  const sliceA = fromPoints([[0, 0], [1, 0], [1, 1]])
+  const sliceB = fromPoints([[0, 1], [1, 0], [1, 1]])
+  const sliceC = fromPoints([[0, 0], [1, 0], [1, 1], [0, 0]])
+
+  t.is(equals(sliceA, sliceA), true)
```

Comment:
Ditto on the equality test

Reply:
Same comment as above. There is no "order" of edges, so the logic needs to change or this function should be dropped.

Reply:
Actually, this is being used by extrudeBySlices to determine if initial and final slices are equal, and adding another segment if not equal.

Reply:
Tests have been simplified.